snuba/cli/replacer.py

```diff
@@ -25,11 +25,11 @@
     help="Kafka bootstrap server to use.",
 )
 @click.option(
-    "--dataset",
-    "dataset_name",
+    "--storage",
+    "storage_name",
     default="events",
-    type=click.Choice(["events", "events_migration"]),
-    help="The dataset to consume/run replacements for (currently only events supported)",
+    type=click.Choice(["events", "errors"]),
+    help="The storage to consume/run replacements for (currently only events supported)",
```

Comment:
What's the rollout plan for production? Now we are passing the dataset name in prod. This will fail upon deployment.

Reply:
Hmm, I thought this wasn't been passed explicitly, but it seems it is for the new errors table. Will need to change this to support both temporarily then.

Reply:
Updated to use the storage from dataset if it is provided